### ISSUE TITLE
`mask` css style not working on Box

### ISSUE THREAD

**Issue by devcer** [chakra-ui/chakra-ui#8174]
### Description

Looks like the css `mask` property is not working on Chakra UI Box. Instead of applying as style, the `mask` property is added as element attribute.

`<div data-testid="star-ratings-star" class="css-1kid1tm" mask="url(&quot;https://beta.heymarvin.com/static/img/star.svg&quot;) center center / contain no-repeat"></div>` 

### Problem Statement/Justification

The [mask](https://developer.mozilla.org/en-US/docs/Web/CSS/mask) property has several use cases but it looks like it isn't working on Box.

### Proposed Solution or API

`mask` CSS style should be supported.

**Comment by segunadebayo** [comment]
Yes, this is not support atm. Please pass it via the `style` prop
